### PR TITLE
Added secondary sort by displayName to RecentlyViewedProjectsService

### PR DESCRIPTION
**Pull request by dtaylor113**
Added secondary sort for 'displayName' (case insensitive).

### CODE REVIEW

File: src/services/recentlyViewedProjects.js [hunk 1]
```diff
@@ -47,17 +47,28 @@ angular.module("openshiftCommonServices")
         }
       });
 
-      // order by creationDate
-      projects =  $filter('orderObjectsByDate')(projects, true);
+      if (secondSort === 'displayName') {
```
Comment (spadgett) [inline src/services/recentlyViewedProjects.js]: I'd rather just always use display name. Do we have a place where we need creation date? We should be consistent everywhere if possible.

**Comment by dtaylor113** [comment]
> Do we have a place where we need creation date?

Yes, the Project Summary side panel is sorted by creation date.

**Comment by spadgett** [comment]
> Yes, the Project Summary side panel is sorted by creation date.

I don't see a reason to sort them differently. I think it should be most recent, and then fall back to display name everywhere.

File: src/services/recentlyViewedProjects.js (hunk 1 above)
Comment (dtaylor113) [inline src/services/recentlyViewedProjects.js]: Fixed

**Comment by dtaylor113** [comment]
Hi @spadgett, ok, changed it so that secondary sort is always case insensitive displayName

**Comment by spadgett** [comment]
Thanks @dtaylor113 LGTM

**Pull request merged**